Makefile

```diff
@@ -2,6 +2,13 @@
 
 TYPE='AUTHORITATIVE'
 
+DISTRO = fedora-26-x86_64
+VERSION = 9
+DG = /home/rpitonak/.local/bin/dg
```

Comment:
This is wrong:( I don't have a home directory called rpitonak. I would prefer to use `/usr/bin/dg` which is provided by distgen package from COPR.